docs/gateway-api/gateway-api-v1.md

```diff
@@ -228,22 +250,30 @@ AKO accepts the following Gateway configuration for this release:
   2. Gateway MUST NOT contain protocols other than HTTP or HTTPS.
   3. Gateway MUST contain a hostname. Hostname as `*` is not supported and `*.domain` is supported.
   4. Gateway MUST NOT contain TLS modes other than `Terminate`.
+  5. If the Secret specified in Gateway-> listeners-> TLSConfig does not exist or is in a namespace other than the Gateway namespace, AKO Gateway implementation will attach `SystemDefaultCert` to the Gateway. Once the Secret is created in the same namespace as that of Gateway, the `certRef` will be updated accordingly.
+  6. Gateway CAN have multiple listeners with same/overlapping hostname. 
+  7. Two Gateways MUST NOT have listeners with same/overlapping hostname.
+  8. `Gateway-> listeners-> allowedRoutes-> namespaces-> from` with value `selector` is not supported.
+  9. `Gateway-> listeners-> allowedRoutes-> namespaces-> selector` is not supported.
+  
 
 #### HTTPRoute Limitations
 
 AKO accepts the following HTTPRoute configuration for this release:
 
   1. HTTPRoute MUST contain at least one parent reference.
   2. HTTPRoute MUST NOT contain `*` as hostname.
-  3. HTTPRoute MUST contain at least one hostname match with parent Gateway
+  3. HTTPRoute MUST NOT contain `*` in hostname.
+  4. HTTPRoute MUST contain at least one hostname match with parent Gateway
+  5. Filters nested inside BackendRefs are not supported i.e. HTTPRoute-> spec-> rules-> backendRefs-> filters are not supported whereas HTTPRoute-> spec-> rules-> filters is supported.
 
 #### Resource Creation
 
-For the Tech preview, AKO imposes a restriction on the order of GatewayAPI object creation. An object that is referenced must be created first. For example, GatewayClass must be created before Gateway and Gateway before HTTPRoute creation. This restriction is only applicable to the Gateway API objects and will be removed in the future releases.
+Currently, AKO imposes a restriction on the order of GatewayAPI object creation. An object that is referenced must be created first. For example, GatewayClass must be created before Gateway and Gateway before HTTPRoute creation. This restriction is only applicable to the Gateway API objects and will be removed in the future releases.
 
 #### High Availability Support
 
-For the Tech Preview, High Availability is not supported.
+Currently, High Availability is not supported.
```

Comment:
This is does not look good, saying high availability is not supported. Replace with: AKO in active-stand-by mode does not support gateway APIs. Avi continues to deliver highly available L4 and L7 LBs for gateway APIs.

Reply:
Done